### ISSUE TITLE
Ubuntu packaging is missing python3-gi-cairo dependency

### ISSUE THREAD

**Issue by rhelmot** [projecthamster/hamster#722]
I installed hamster on a brand new ubuntu 23.04 system through apt and it failed to launch:

```
Traceback (most recent call last):
  File "/bin/hamster", line 149, in on_activate_window
    self._open_window(action.get_name(), data)
  File "/bin/hamster", line 184, in _open_window
    self.overview_controller = Overview()
                               ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/hamster/overview.py", line 451, in __init__
    self.fact_tree = FactTree()
                     ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/hamster/widgets/facttree.py", line 279, in __init__
    self.date_label = Label(10, 3)
                      ^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/hamster/widgets/facttree.py", line 57, in __init__
    self.layout = pangocairo.create_layout(self._label_context)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'could not find foreign type Context'
```

Installing python3-gi-cairo fixed the issue.

I'm not sure if this is the right place to report packaging issues, but I'd gladly take a pointer to such a place if this isn't it!

**Comment by GeraldJansen** [comment]
@rhertzog I reported this here: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1036661.

**Comment by matthijskooijman** [comment]
Note that this was recently already fixed in the README in this repo: #703. Since there's nothing more to do *here*, and the report has been forwarded (thanks @GeraldJansen) to Debian (where the Ubuntu package comes from), so I'll go ahead and close this issue.
